test/persistence-hooks.suite.js

```diff
@@ -2014,6 +2014,7 @@ module.exports = function(dataSource, should, connectorCapabilities) {
             if (dataSource.connector.updateOrCreate) {
```

Comment:
AFAICT, both branches report the same context data now and this `if` can be removed.